sphinx/ext/imgmath.py

```diff
@@ -76,7 +76,7 @@ class InvokeError(SphinxError):
 '''
 
 DOC_BODY_PREVIEW = r'''
-\usepackage[active]{preview}
+\usepackage[active,tightpage]{preview}
```

Comment:
This is needed for dvisvgm to gather "depth" information. I hope it does not affect dvipng usage (with its -Ttight option).

Reply:
In fact adding this option did change the dvipng console output and thus broke imgmath for it because the regex did not find matches anymore. Fixed in second commit.

Reply:
We probably don't need to use ``-T tight`` option of dvipng anymore.

Reply:
The ``tightpage`` option does not play well with ``dvipng``. In my latest commit I am thus using it *only*  for ``dvisvgm``. As per my previous comment, the ``-T tight`` option of ``dvipng`` remains required.